libs/activeObject/tests/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+liblet_tests(
+  SOURCES
+    activeObjectTest.cpp
```

Comment:
Add this test (and others you've created) to the CI pipelines. The tests are kicked off at the end of these two files:

https://github.com/microsoft/Mso/blob/master/.ado/templates/posix-build-steps.yml
https://github.com/microsoft/Mso/blob/master/.ado/templates/windows-build-job.yml

Reply:
I will do. Thank you!